src/main/java/de/fraunhofer/aisec/codyze/config/CodyzeConfiguration.kt

```diff
@@ -69,3 +59,65 @@ class AnalysisMode {
     )
     var tsMode = TypestateMode.DFA
 }
+
+class MarkArgGroup {
+    var append = false
+    var matched = false
+
+    var mark: Array<File> = arrayOf(File("./"))
+
+    @Option(
+        names = ["-m", "--mark"],
+        paramLabel = "<path>",
+        description = ["Loads MARK policy files.\n\t(Default: \${DEFAULT-VALUE})"],
+        split = ","
+    )
+    fun match(value: Array<File>) {
+        matched = true
+        this.mark = value
+    }
+
+    @Option(
+        names = ["--mark+"],
+        paramLabel = "<path>",
+        description =
+            ["See --mark, but appends the values to the ones specified in configuration file."],
+        split = ","
+    )
+    fun append(value: Array<File>) {
+        append = true
+        match(value)
+    }
+}
+
+class DisabledMarkArgGroup {
+    var append = false
+    var matched = false
+
+    var disabledMarkRules: List<String> = emptyList()
+    @Option(
+        names = ["--disabled-mark-rules"],
+        paramLabel = "<package.rule>",
+        description =
+            [
+                "The specified mark rules will be excluded from being parsed and processed. The rule has to be specified by its fully qualified name (package.rule). If there is no package name, specify rule as \".rule\". Use \'*\' to disable an entire package."],
+        split = ","
+    )
+    fun match(value: List<String>) {
+        matched = true
+        this.disabledMarkRules = value
+    }
+
+    @Option(
+        names = ["--disabled-mark-rules+"],
+        paramLabel = "<package.rule>",
+        description =
+            [
+                "See --disabled-mark-rules, but appends the values to the ones specified in configuration file."],
+        split = ","
```

Comment:
Do we continue using `,` or should we switch to system agnostic path separators?